### ISSUE TITLE
Downloading Wordpress version undefined

### ISSUE THREAD

**Issue by trikon** [romainberger/yeoman-wordpress#30]
Windows 7 x64, 
node - v0.10.15,
npm - 1.3.5,
yo - 1.0.0-rc.1.3

```
D:\projects\yo\wp>yo yo-wordpress


Trying to get the latest stable version of Wordpress
[?] Name of the theme you want to use: (mytheme) Let's download the framework, shall we?
Downloading Wordpress version undefined
     info ... Fetching https://github.com/WordPress/WordPress/tarball/undefined ...
     info This might take a few moments
.
? Done in 1.65s.
unzip error { [Error: incorrect header check] errno: -3, code: 'Z_DATA_ERROR' }

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: incorrect header check
    at Zlib._binding.onerror (zlib.js:295:17)
```

strange that fetching branch is undefined.

**Comment by romainberger** [comment]
Hi,
Someone had a similar issue on Windows, apparently using gitbash helped (sorry I don't use windows and have no idea what this is :confused: ). The issue is here if you want to see exactly what happened => https://github.com/romainberger/yeoman-wordpress/issues/27.

Let me know if you still have errors I'll try to see what I can do

**Comment by romainberger** [comment]
Fixed in cb74082cbfa86e6b07206c95f5fa16759b29867a
